README.md

```diff
@@ -58,6 +58,14 @@ If you want to set host to a specific IP address for example `172.17.12.1` run `
 
 *To view visit http://localhost:8000/demos/<name>*
 
+## Guidelines for CSS Development
+
+- For issues created in Core that will affect a component in PF-React, a follow up issue must be created in PF-React once the Pull Request is merged.
+- The UX/Visual designer and CSS developer of a component/enhancement should be involved in the review process throughout the life cycle of that component/enhancement.
+- If global variables are modified in Core, a React issue should be opened to address this.
+- CSS developers should ensure that animation is well documented and communicated to the respective React developer.
```

Comment:
I think we could probably expand on this once we have a better sense of how we will tackle animation in core.

Reply:
Should I leave this in for now, or wait until we discuss animation further.

Reply:
ya I think it's ok to leave for now.